cmd/lncli/commands.go

```diff
@@ -1183,10 +1188,17 @@ func abandonChannel(ctx *cli.Context) error {
 // line. Both named options as well as unnamed parameters are supported.
 func parseChannelPoint(ctx *cli.Context) (*lnrpc.ChannelPoint, error) {
 	channelPoint := &lnrpc.ChannelPoint{}
+	var err error
 
 	args := ctx.Args()
 
 	switch {
+	case ctx.IsSet("chan_point"):
+		channelPoint, err = parseChannelPointStr(ctx.String("chan_point"))
+		if err != nil {
+			return nil, fmt.Errorf("unable to parse chan_point: %v", err)
+		}
+		return channelPoint, nil
```

Comment:
nit: add empty line after return to visually space the `case` statements a bit.

Reply:
done